contrib/freshmaker-cli

```diff
@@ -224,6 +229,45 @@ def get_freshmaker_events_by_ids(api_url, ids):
     events = sorted(events, key=lambda k: k['id'])
     return events
 
+def get_advisories_from_errata():
+    """
+    Sends the GET request to the Errata filter for the latest RHSA, in SHIPPED state. Returns the
+    list of advisories that are IMPORTANT or CRITICAL.
+
+    """
+    krb_auth = HTTPKerberosAuth()
+    r = requests.get("https://errata.devel.redhat.com/filter/2943.json", verify=False, auth=krb_auth)
+    r.raise_for_status()
+    data = r.json()
+    advisories = []
+    for advisory in data:
+        if advisory['security_impact'] == "Important" or advisory['security_impact'] == "Critical":
+            advisories.append(advisory['id'])
+    return advisories
+
+
+def get_freshmaker_events_by_search_keys(api_url, search_keys):
+    print("Getting Freshmaker event id..")
+    events = []
+    for i in search_keys:
+        url = '%s/api/1/events/?search_key=%s' % (api_url, i)
+
+        r = requests.get(url, verify=False)
```

Comment:
Why use `verify=False`?

Why not use request session?

Reply:
I have the SSL error without it

Reply:
This is not needed. This is likely because you installed python-requests from pip instead of the Fedora/Red Hat repos. This means that it doesn't use the certificate authorities trusted on your system to determine if the certificate can be trusted. As a work-around, you can run the command with this environment variable set:
```
REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-bundle.crt
```